wranglertools/get_field_info.py

```diff
@@ -281,8 +295,8 @@ def dotted_field_name(field_name, parent_name=None):
         return field_name
 
 
-def build_field_list(properties, required_fields=None, include_description=False,
-                     include_comment=False, include_enums=False, parent='', is_submember=False, admin=False):
+def build_field_list(properties, required_fields=None, no_description=False,
+                     include_comment=False, no_enums=False, parent='', is_submember=False, admin=False):
```

Comment:
`no_description` and `no_enums` are now used with the opposite meaning as `include_description` and `include_enums`, but the default `False` value is not changed. I think it makes sense, but just want to double-check that this is intended.

Reply:
Yes, I think now the defaults used (even though I believe they will always be passed in from values assigned elsewhere) make sense for the normal use case so are OK as specified.